src/styles/schema-styles.js

```diff
@@ -58,6 +58,7 @@ export default css`
 
 .stri, .string, .uri, .url, .byte, .bina, .date, .pass, .ipv4, .ipv4, .uuid, .emai, .host {color:var(--green);}
 .inte, .numb, .number, .int6, .int3, .floa, .doub, .deci .blue {color:var(--blue);}
+.formnum {color:var(--orange);}
```

Comment:
Let's not change the color in this PR, we can discuss in a follow up once we fix the array formatting.

Reply:
![Screen Shot 2022-09-02 at 1 49 02 PM](https://user-images.githubusercontent.com/28315000/188219541-b8810bb7-4c4b-42fa-aab3-3541ba0b7eca.png)

Reply:
What are you trying to show me here?

Reply:
nothing more. 
Look like i wasting time in useless discussions that most not about code quality, but about how to keep tool not informative, because some good solution will remind about Swagger. :)
You can take "Swagger Petstore - OpenAPI 3.0" and test your tool. 
Yes, this is from Swagger, but to check API that built not by you will give you pile of experience about info that your tool provide and what not.
Let me know if i can somehow help

Reply:
It's about context. I appreciate the PR the problem is that we aren't on the same page yet. You have things that you see, and you are trying to fix them but without sharing the context. It would really help instead of a PR to start with the issue and then we can go from there:
* Here is problem X
* Here is problem Y
Of which we can talk about each one in the PR and figure out the best solution for everyone.

Right now we are talking about the solution before agreeing about the problem.